include/sound/sof/header.h

```diff
@@ -76,6 +76,7 @@
 #define SOF_IPC_PM_CLK_REQ			SOF_CMD_TYPE(0x006)
 #define SOF_IPC_PM_CORE_ENABLE			SOF_CMD_TYPE(0x007)
 #define SOF_IPC_PM_GATE				SOF_CMD_TYPE(0x008)
+#define SOF_IPC_PM_VFE_RESET_STATUS		SOF_CMD_TYPE(0x010)
```

Comment:
typo in the commit message "hast last time"

Reply:
thanks, fixing